import-scripts/dmp-import-vars-functions.sh

```diff
@@ -90,6 +90,13 @@ function sendImportSuccessMessageMskPipelineLogsSlack {
     curl -X POST --data-urlencode "payload={\"channel\": \"#msk-pipeline-logs\", \"username\": \"cbioportal_importer\", \"text\": \"MSK cBio pipelines import success: $STUDY_ID\", \"icon_emoji\": \":tada:\"}" $SLACK_PIPELINES_MONITOR_URL
 }
 
+# Function for alerting slack channel of clear cache failures
+function sendClearCacheFailureMessageMskPipelineLogsSlack {
+    CACHE_GROUP_NAME=$1
+    SOURCE_SCRIPT_NAME=$2
+    curl -X POST --data-urlencode "payload={\"channel\": \"#msk-pipeline-logs\", \"username\": \"cbioportal_importer\", \"text\": \"Failure to clear the persistence cache for $CACHE_GROUP_NAME portals during import script $SOURCE_SCRIPT_NAME\", \"icon_emoji\": \":plunger:\"}" $SLACK_PIPELINES_MONITOR_URL
```

Comment:
unless we're really planning to monitor processes to this extent, I think this might just add to noise in the slack channel.

Reply:
this was changed to a mail message to cbioportal-pipelines